full/src/test/java/apoc/ApocSplitTest.java

```diff
@@ -0,0 +1,631 @@
+        "apoc.cypher.runSchemaFiles",
+        "apoc.cypher.parallel",
+        "apoc.cypher.mapParallel",
+        "apoc.cypher.mapParallel2",
+        "apoc.cypher.parallel2",
+        "apoc.gephi.add",
+        "apoc.mongo.aggregate",
+        "apoc.mongo.count",
+        "apoc.mongo.find",
+        "apoc.mongo.insert",
+        "apoc.mongo.update",
+        "apoc.mongo.delete",
+        "apoc.mongodb.get.byObjectId",
+        //"apoc.load.xls",
+        "apoc.load.csv",
+        "apoc.load.csvParams",
+        "apoc.load.ldap",
+        "apoc.load.driver",
+        "apoc.load.jdbc",
+        "apoc.load.jdbcUpdate",
+        "apoc.load.htmlPlainText",
+        "apoc.load.html",
+        "apoc.load.directory.async.add",
+        "apoc.load.directory.async.remove",
+        "apoc.load.directory.async.removeAll",
+        "apoc.load.directory.async.list",
+        "apoc.load.directory",
+        "apoc.dv.catalog.add",
+        "apoc.dv.catalog.remove",
+        "apoc.dv.catalog.list",
+        "apoc.dv.query",
+        "apoc.dv.queryAndLink",
+        "apoc.model.jdbc",
+        "apoc.generate.ba",
+        "apoc.generate.ws",
+        "apoc.generate.er",
+        "apoc.generate.complete",
+        "apoc.generate.simple",
+        "apoc.log.error",
+        "apoc.log.warn",
+        "apoc.log.info",
+        "apoc.log.debug",
+        "apoc.es.stats",
+        "apoc.es.get",
+        "apoc.es.query",
+        "apoc.es.getRaw",
+        "apoc.es.postRaw",
+        "apoc.es.post",
+        "apoc.es.put",
+//        "apoc.export.xls.all",
+//        "apoc.export.xls.data",
+//        "apoc.export.xls.graph",
+//        "apoc.export.xls.query",
+        "apoc.custom.declareProcedure",
+        "apoc.custom.declareFunction",
+        "apoc.custom.list",
+        "apoc.custom.removeProcedure",
+        "apoc.custom.removeFunction",
+        "apoc.uuid.install",
+        "apoc.uuid.remove",
+        "apoc.uuid.removeAll",
+        "apoc.uuid.list",
+        "apoc.couchbase.get",
+        "apoc.couchbase.exists",
+        "apoc.couchbase.insert",
+        "apoc.couchbase.upsert",
+        "apoc.couchbase.append",
+        "apoc.couchbase.prepend",
+        "apoc.couchbase.remove",
+        "apoc.couchbase.replace",
+        "apoc.couchbase.query",
+        "apoc.couchbase.posParamsQuery",
+        "apoc.couchbase.namedParamsQuery",
+        "apoc.nlp.azure.entities.graph",
+        "apoc.nlp.azure.entities.stream",
+        "apoc.nlp.azure.keyPhrases.graph",
+        "apoc.nlp.azure.keyPhrases.stream",
+        "apoc.nlp.azure.sentiment.graph",
+        "apoc.nlp.azure.sentiment.stream");
+
+    public static final List<String> FULL_FUNCTIONS = List.of(
+        "apoc.trigger.nodesByLabel",
+        "apoc.trigger.propertiesByKey",
+        "apoc.trigger.toNode",
+        "apoc.trigger.toRelationship",
+        "apoc.ttl.config",
+        "apoc.static.get",
+        "apoc.static.getAll",
+        "apoc.coll.avgDuration"
+        //"apoc.data.email"
```

Comment:
See [this](https://github.com/neo4j-contrib/neo4j-apoc-procedures/pull/3079/commits/53da905ca870041d26a60c966f3c3e7803ce164f#r933362761) comment